test/checkout.test.js

```diff
@@ -69,7 +69,24 @@ describe('Shopify#checkout', () => {
 
     scope
       .post(`/admin/checkouts/${token}/complete.json`, {})
-      .reply(202, output);
+      .reply(200, output);
+
+    return shopify.checkout.complete(token)
+      .then(data => expect(data).to.deep.equal(output.checkout));
+  });
+
+  it('completes a free checkout and retries 202 response', () => {
+    const token = 'b490a9220cd14d7344024f4874f640a6';
+    const baseUrl = shopify.baseUrl;
+    const pathname = `/admin/checkouts/${token}/complete.json`;
+    const href = `${baseUrl.protocol}//${baseUrl.hostname}${pathname}`;
+    const output = fixtures.res.complete;
+
+    scope
+      .post(pathname, {})
+      .reply(202, output, { Location: href, 'Retry-After': 0 })
```

Comment:
```suggestion
      .reply(202, '', { Location: href, 'Retry-After': 0 })
```

Shouldn't body be empty?